File: prak3/disk.java
class Disk{
    //atribut
    private String type;
    private int capacity;
    private int price;
     
    //konstruktor 
    Disk(){}

    //setter & getter type
    public void settype(String type){
        this.type = type;
    }
    public String gettype(){
        return type;
    }

    //setter & getter capacity
    public void setcapacity(int capacity){
        this.capacity = capacity;
    }
    public int getcapacity(){
        return capacity;
    }

    //setter & getter price
    public void setprice(int price){
        this.price = price;
    }
    public int getprice(){
        return price;
    }

    void printDisk(){
        System.out.println("Disk        : " + this.type + " " + this.capacity + " GB");
    }
}

File: python/disk.py
class Disk():
    #private members
    __type = "blank"
    __capacity = 0
    __price = 0

    def __init__(self, type = "blank", capacity = 0, price = 0):
        self.__type = type
        self.__capacity = capacity
        self.__price = price
       

    #getter & setter
    def setcapacity(self, capacity):
        self.__capacity = capacity

    def getcapacity(self):
        return self.__capacity
    
    def setprice(self, price):
        self.__price = price

    def getprice(self):
        return self.__price

    def settype(self, type):
        self.__type = type

    def gettype(self):
        return self.__type

    def printDisk(self):
        print("Disk        : " + str(self.gettype()) + " " + str(self.getcapacity()) + " GB")  
